go.mod

```diff
@@ -9,38 +9,43 @@ require (
 	github.com/cpuguy83/go-md2man v1.0.10
 	github.com/docker/cli v20.10.8+incompatible
 	github.com/docker/docker v20.10.8+incompatible
-	github.com/fatih/color v1.9.0
+	github.com/fatih/color v1.12.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/google/go-cmp v0.5.6
-	github.com/google/go-containerregistry v0.5.2-0.20210709161016-b448abac9a70
+	github.com/google/go-containerregistry v0.6.1-0.20210922191434-34b7f00d7a60
 	github.com/hako/durafmt v0.0.0-20191009132224-3f39dc1ed9f4
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
-	github.com/jonboulle/clockwork v0.1.1-0.20190114141812-62fb9bc030d1
+	github.com/jonboulle/clockwork v0.2.2
 	github.com/ktr0731/go-fuzzyfinder v0.2.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
+	github.com/tektoncd/chains v0.5.1-0.20211019171834-eee44752460a
 	github.com/tektoncd/hub/api v0.0.0-20210922050526-aa589697cb58
 	github.com/tektoncd/pipeline v0.28.0
-	github.com/tektoncd/plumbing v0.0.0-20210514044347-f8a9689d5bd5
+	github.com/tektoncd/plumbing v0.0.0-20210902122415-a65b22d5f63b
 	github.com/tektoncd/triggers v0.16.0
 	go.opencensus.io v0.23.0
 	go.uber.org/multierr v1.7.0
-	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
+	go.uber.org/zap v1.19.1
+	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools v2.2.0+incompatible
 	gotest.tools/v3 v3.0.3
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
 	k8s.io/cli-runtime v0.21.4
-	k8s.io/client-go v0.21.4
-	knative.dev/pkg v0.0.0-20210827184538-2bd91f75571c
-	sigs.k8s.io/yaml v1.2.0
+	k8s.io/client-go v0.22.1
+	knative.dev/pkg v0.0.0-20211004133827-74ac82a333a4
+	sigs.k8s.io/yaml v1.3.0
 )
 
 replace (
 	github.com/kr/pty => github.com/creack/pty v1.1.16
+	github.com/tektoncd/chains => /Users/rgreinhofer/projects/tektoncd/chains
```

Comment:
@rgreinho you need to remove this for the CI (and for use to test) to work 👼🏼

Reply:
Ah yes, sorry about that. I just converted the PR back to draft. We're still working on https://github.com/tektoncd/chains/pull/284, which should be required before moving forward with this one.

Reply:
ah ok 👌🏼